src/services/mongoose/users.ts

```diff
@@ -57,13 +78,23 @@ export const UsersRepository = (): IUsersRepository => {
     }
   }
 
+  const updateIps = async (id, ips) => {
```

Comment:
See above

Reply:
we only want to update the ip array.because we run this function asynchronously, we don't want to the other elements of the user. with the different promise running in parrallel, we would be at a risk of writing stale data if we were to use the more generic `update` function below